### PR TITLE
Fix belongs_to tag value to support underscored.

### PR DESCRIPTION
**Pull request by larrymjordan**
This PR solves https://github.com/gobuffalo/pop/issues/469 and allows consistency with the way `fk_id` is defined in `has_many` and `many_to_many`.

This PR also allows support for pointers in fields that defines `belongs_to`.

### CODE REVIEW

File: associations/belongs_to_association.go [hunk 1]
```diff
@@ -31,7 +31,22 @@ func belongsToAssociationBuilder(p associationParams) (Association, error) {
...
+	ownerIDField := fmt.Sprintf("%s%s", p.field.Name, "ID")
+
+	if tags.Find("fk_id").Value != "" {
+		dbTag := tags.Find("fk_id").Value
+		if _, found := p.modelType.FieldByName(dbTag); !found {
+			t := p.modelValue.Type()
+			for i := 0; i < t.NumField(); i++ {
+				f := t.Field(i)
+				if f.Tag.Get("db") == dbTag {
+					ownerIDField = f.Name
```
Comment (stanislas-m) [inline associations/belongs_to_association.go:43]: Why do you need to continue after finding a match? :)

Comment (larrymjordan) [reply associations/belongs_to_association.go:43]: you are right, I missed this part, thank you for taking a look at it.

**Comment by stanislas-m** [comment]
Thanks @larrymjordan!

**Comment by aeneasr** [comment]
Awesome, thank you!

**Pull request merged**